### PR TITLE
xhamster3.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [X] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/136215#issuecomment-1333715890

### Add your comment and screenshots

None.

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/specific.txt [hunk 1]
```diff
@@ -275,6 +275,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+||*.xyz^$domain=xhamster16.*|hamsterix.*|xhamster13.*|xhamster18.*|xhamster19.*|xhamster11.*|xhamster5.*|xhamster10.*|xhamster4.*|xhamster3.*|xhamster2.*|xhamster.*
```
Comment (Yuki2718) [inline BaseFilter/sections/specific.txt:278]: Why not `||xyz^$domain=` ? Also I think better to limit to subdocument.

Comment (piquark6046) [reply BaseFilter/sections/specific.txt:278]: Are all mentioned request advertisement-related?

Comment (Alex-302) [reply BaseFilter/sections/specific.txt:278]: Hm. I don't see requests to `.xyz`

Comment (piquark6046) [reply BaseFilter/sections/specific.txt:278]: Please click one of videos.
@Alex-302

Comment (Yuki2718) [reply BaseFilter/sections/specific.txt:278]: > Are all mentioned request advertisement-related?

Yes.

Comment (piquark6046) [reply BaseFilter/sections/specific.txt]: Pushed.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**